IdentityCore/src/network/error_handler/MSIDAADRequestErrorHandler.m

```diff
@@ -44,25 +44,27 @@ - (void)handleError:(NSError *)error
     BOOL shouldRetry = YES;
     shouldRetry &= httpRequest.retryCounter > 0;
     BOOL shouldRetryNetworkingFailure = NO;
```

Comment:
nit: line 46 can be moved to line 49 as it is not needed in the outer scope :)

Reply:
updated